### PR TITLE
[stable/nginx-ingress] Add fine-grained host port range in psp

### PR DESCRIPTION
**Pull request by secret104278**
Give port-by-port permission in psp to improve security.

Signed-off-by: Yu Chih <secret104278@gmail.com>

No

#### What this PR does / why we need it:
When `podSecurityPolicy` and `hostNetwork` is enable, give `hostPort` permission on demain.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/nginx-ingress/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.27.1
+version: 1.27.2
```
Comment (ChiefAlexander) [inline stable/nginx-ingress/Chart.yaml]: Let's bump the minor version to `1.28.0`

**Review by ChiefAlexander — CHANGES_REQUESTED** [review]

File: stable/nginx-ingress/Chart.yaml (hunk 1 above)
Comment (secret104278) [reply stable/nginx-ingress/Chart.yaml]: OK

**Pull request merged**